parts/language/dict_en.lua

```diff
@@ -224,45 +224,45 @@ FNNS and     {"Support 3",
     {"Worldwide Combos",
         "wwc worldwidecombos",
         "game",
-        "Browser | Singleplayer/Multiplayer\nWWC for short. It has worldwide 1V1 battles, recorded battles (so the opponents don’t have to be real people), several different rulesets, and bomb-styled garbage line battles.",
+        "Browser | Singleplayer/Multiplayer\nWWC for short. It has worldwide 1V1 battles, recorded battles (which allows you to battle against recorded games), several different rulesets, and bomb-style garbage line battles.",
         "https://worldwidecombos.com",
     },
     {"Tetris Friends",
         "tf tetrisfriends notrisfoes",
         "game",
-        "Browser | Singleplayer/Multiplayer\nTF for short. A now-defunct browser-based Tetris game. It was very popular in the past, but the whole game was shut down a few years ago. There is a private server called “Notris Foes” that still exists. Click on the globe icon to open the webpage in your browser.",
+        "Browser | Singleplayer/Multiplayer\nTF for short. A defunct browser-based Tetris game. The game was very popular before shutting down a few years ago. There is a private server called \"Notris Foes\" that still exists. Click on the globe icon to open the webpage.",
         "https://notrisfoes.com",
     },
     {"tetris.com",
         "tetris online official",
         "game",
-        "Browser | Singleplayer\nThe official Tetris game on tetris.com, with only one mode (marathon). It supports a smart, mouse-based control system.",
+        "Browser | Singleplayer\nThe official Tetris game on tetris.com, with only one mode (marathon). It supports a mouse-based \"smart\" control system.",
```

Comment:
We should add ``Mobile support`` since this game is now playable on mobile. And add it to Tetris Gems and Tetris Mind Bender too

Reply:
A browser game implies mobile support here (unless an entry explicitly says no).